### ISSUE TITLE
Comment: Creating the application (guides/chat/creating.md)

### ISSUE THREAD

**Issue by adnanusman** [feathersjs/docs#1277]
### Comment/Problem

Tell us about your issue or comment. Please be as specific as possible.

I ran the feathers generate app function, then ran npm test.

I saw this error:

'NODE_ENV' is not recognized as an internal or external command,
operable program or batch file.

Everything after this seems to work fine, this is a pretty amazing thing you guys made! Thank you!

If you would like to point out a typo, wording or code issue, please submit a pull request with a suggested fix instead by following the "Edit this page" link.

### System configuration (optional)

Tell us about the applicable parts of your setup.

**Module versions** (especially the part that's not working):
  "dependencies": {
    "@feathersjs/configuration": "^2.0.6",
    "@feathersjs/errors": "^3.3.6",
    "@feathersjs/express": "^1.3.1",
    "@feathersjs/feathers": "^3.3.1",
    "@feathersjs/socketio": "^3.2.9",
    "compression": "^1.7.3",
    "cors": "^2.8.5",
    "helmet": "^3.16.0",
    "serve-favicon": "^2.5.0",
    "winston": "^3.0.0"
  },
  "devDependencies": {
    "eslint": "^5.15.1",
    "mocha": "^5.2.0",
    "nodemon": "^1.18.10",
    "request": "^2.88.0",
    "request-promise": "^4.2.2"
  }

**NodeJS version**:
v8.12.0

**Operating System**:
I am using Windows 10.

**Browser Version**:
Chrome 72.0.3626.121 (Official Build) (64-bit)

**React Native Version**:
N/A

**Module Loader**:
npm

**Comment by timcwilson** [comment]
I edited out NODE_ENV from the test line of the script - seems more an issue with the generator. I don't know enough to tell if this will be an issue testing production.

Also I had to add:
 "devDependencies": {
    "babel-eslint": "^10.0.1",
    "eslint": "^5.16.0",
    "eslint-config-airbnb": "^17.1.0",
    "eslint-plugin-import": "^2.17.2",
    "eslint-plugin-jsx-a11y": "^6.2.1",
    "eslint-plugin-react": "^7.12.4",

Before I could run "npm test" properly.
And again the generator files had the following three issues:
\src\app.js
  6:16  error  `./logger` import should occur after import of `@feathersjs/socketio`  import/order
\src\hooks\log.js
  5:14  error  `util` import should occur before import of `../logger`  import/order
\src\index.js
  3:1  error  Expected 1 empty line after require statement not followed by another require  import/newline-after-import

After running the tests with --fix option, then all tests passed.
